modules/nf-core/bbmap/bbmerge/main.nf

```diff
@@ -0,0 +1,59 @@
+process BBMAP_BBMERGE {
+    tag "$meta.id"
+    label 'process_single'
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bbmap:39.06--h92535d8_0':
+        'biocontainers/bbmap:39.06--h92535d8_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+
+    output:
+    tuple val(meta), path("*_merged.fastq"), emit: merged
+    tuple val(meta), path("*_unmerged.fastq"), emit: unmerged
+    tuple val(meta), path("*_ihist.txt"), emit: ihist
+    path "versions.yml"           , emit: versions
```

Comment:
Can you update this to follow the "Harshil Alignment" format? 
😄 

https://nf-co.re/docs/contributing/modules#what-is-the-harshil-alignment